Dockerfile

```diff
@@ -39,14 +39,15 @@ RUN apt-get update && apt-get install  -y \
     libssl-dev \
     libudev-dev \
     libwayland-dev \
-    libwebkit2gtk-4.0-dev \
     libxkbcommon-dev \
     locales \
     locales-all \
     m4 \
     pkgconf \
     sudo \
     wayland-protocols \
+    bash \
```

Comment:
You shouldn't need to install bash.  Not a big deal, but you could snip this if you want.